man/fi_msg.3.md

```diff
@@ -222,7 +222,7 @@ fi_sendmsg.
 
 *FI_CLAIM*
 : Applies to posted receive operations for endpoints configured
-  for FI_BUFFERED_RECV or FI_VARIABLE_MSG.  This flag is used to
+  for FI_BUFFERED_RECV.  This flag is used to
```

Comment:
In addition to the `FI_VARIABLE_MSG` flag, the `FI_BUFFERED_RECV` flag was _also_ removed in this pull request. Therefore, shouldn't the `FI_CLAIM` and `FI_DISCARD` flags _also_ be removed?

Reply:
FI_CLAIM and FI_DISCARD can also be used with FI_PEEK.

Reply:
@mblocksome and @j-xiong are both correct.  However, I agree with Mike.  This change is to the fi_msg man page, and FI_PEEK does not apply there.  FI_CLAIM and FI_DISCARD can be removed as flags for the fi_msg calls.  They remain for fi_tagged.